scheduler/test/cook/test/kubernetes/api.clj

```diff
@@ -95,7 +95,7 @@
           (is (= ["/bin/sh" "-c" "foo && bar"] (.getCommand container)))
           (is (= "alpine:latest" (.getImage container)))
           (is (not (nil? container)))
-          (is (= 4 (count (.getEnv container))))
+          (is (= 5 (count (.getEnv container))))
```

Comment:
Nitpick: can we explicitly also assert on the environment map?

Reply:
Oh, I did add that—forgot to push it!

Reply:
I was doing a pass over the code and fixing multiple small issues, but I wanted to wait until I was all the way through to push all of them—and then I got distracted patching a flaky test in another branch. Sorry!